IHP/View/Types.hs

```diff
@@ -99,6 +101,12 @@ data InputType
     | FileInput
 
 
+data BreadcrumbsView =
+    BreadcrumbsView
+    { cssFramework :: !CSSFramework
```

Comment:
This `cssFramework` is never used, so I think can we remove it?

Reply:
Yes 👍 

What do you think about renaming this data structure to just `Breadcrumb`?

Reply:
Your call, but I think it's better like that so it's consistent with `PaginationView`

Reply:
Actually in this case we don't need this at all, as we already pass `[BreadcrumbItem]`

Reply:
Sorry, we do need it - to pass the `BreadcrumbItem`s as HTML